systemtests/tests/py2plug-fd-ldap/etc/bareos/bareos-dir.d/fileset/PluginTest.conf.in

```diff
@@ -0,0 +1,10 @@
+FileSet {
+  Name = "PluginTest"
+  Description = "Test the Plugin functionality with a Python Plugin."
+  Include {
+    Options {
+      signature = MD5
+    }
+    Plugin = "@python_module_name@:module_path=@python_plugin_module_src_test_dir@:module_name=bareos-fd-ldap:uri=ldap\\://@SYSTEMTEST_LDAP_ADDRESS@:basedn=ou=backup,@SYSTEMTEST_LDAP_BASEDN@:bind_dn=@SYSTEMTEST_LDAP_BINDDN@:password=@SYSTEMTEST_LDAP_PASSWORD@"
```

Comment:
How about using the newly available multi-line strings here?

Reply:
Same as above. Great idea, but we won't be able to backport the test to Bareos 19 as easily if we use a modern Bareos 20 feature here.